suite-native/transactions/src/components/TransactionDetail/TransactionDetailRow.tsx

```diff
@@ -23,7 +23,7 @@ export const TransactionDetailRow = ({ title, children }: TransactionDetailRowPr
         <Box style={applyStyle(rowStyle)}>
             <Text color="textSubdued">{title}</Text>
             <Box flexDirection="row" justifyContent="flex-start">
-                {G.isString(children) ? <Text>{children}</Text> : children}
+                {G.isString(children) || G.isNumber(children) ? <Text>{children}</Text> : children}
```

Comment:
Since `ReactNode` basically turns off type checking, I would recommend to retype it in this instance to just string as that's what you want the type to be. I don't see any usage where you want something else than string (maybe I missed it). That means string would be better type as well as better error handling in component where you use it.

Reply:
The `TransactionDetailRow` was designed to accept both primitive (string/number) and ReactNode children. The ReactNode is used for example in here: https://github.com/trezor/trezor-suite/blob/b4fd53c20ec799883ae1f716b4447110c860efab/suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx#L45-L52